CodeStarterCamp_Week4/main.swift

```diff
@@ -54,10 +69,59 @@ class BodyCondition {
     }
 }
 
-func doExercise(activity: Activity, body: BodyCondition) -> Void {
+func readNumRoutine() throws -> Int {
+    let readData = readLine()
+    
+    guard let data = readData else { throw ExerciseError.incorrectInputError }
+    guard let numData = Int(data) else { throw ExerciseError.incorrectInputError }
```

Comment:
같은 에러를 반환하는 경우이니 이렇게 한 번에 작성할 수도 있습니다!

```suggestion
    guard let data = readData, let numData = Int(data) else { 
        throw ExerciseError.incorrectInputError 
    }
```